test/tools/ossfuzz/protoToYul.h

```diff
@@ -32,6 +33,19 @@ namespace yul_fuzzer
 class ProtoConverter
 {
 public:
+	ProtoConverter()
+	{
+		m_numLiveVars = 10;
```

Comment:
Why does it start with 10?

Reply:
Because of the hard-coded function template that uses up 10 variables in the function parameters.

https://github.com/ethereum/solidity/blob/7c880a26c021fa949f400c9ce5b02a05756bc2a7/test/tools/ossfuzz/protoToYul.cpp#L385

Reply:
Please add a comment about that.

Reply:
done.

Reply:
done